Bottom: d40415b72ef6d25d6bc8bc7b424cbd82b8df8e58
Top:    1dea80a703036b8141e234ea76f2cf07059cf89f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 02:29:45 +0000

configure.ac: Don't add GD_CFLAGS to CFLAGS, only CPPFLAGS

GD_CFLAGS will almost certainly contain pre-processor flags, not strictly
compiler flags. Leave CFLAGS uncluttered so the user may override it sensibly.


---

diff --git a/configure.ac b/configure.ac
index fd16e85..8ed21a0 100644
--- a/configure.ac
+++ b/configure.ac
@@ -791,7 +791,6 @@ you will get lucky.
 	# since some linux systems evidently install gdlib-config but fail to
 	# install the headers (nice), check for the header too and fail if it
 	# is not there.
-	CFLAGS="$CFLAGS $GD_CFLAGS"
 	CPPFLAGS="$CPPFLAGS $GD_CFLAGS"
 	AC_CHECK_HEADERS(gd.h)
 	case $ac_cv_header_gd_h in
